functions/clients/rawg.go

```diff
@@ -24,3 +24,19 @@ func NewRawg(rawgToken string) *Rawg {
 func (r *Rawg) GetGame(ctx context.Context, game string) (*rawg.GameDetailed, error) {
 	return r.client.GetGame(ctx, game)
 }
+
+func RawgErrorMsg(err error) string {
+	rawgError, ok := err.(*rawg.RawgError)
```

Comment:
Not sure I like converting an error to a string. We might not need this function as adding a game will likely be the only place we check for a 404.